dashboard/app/controllers/foorm_editor_controller.rb

```diff
@@ -0,0 +1,21 @@
+class FoormEditorController < ApplicationController
+  # GET '/foorm/editor/'
+  def index
+    # only show for admins on non-production
+    return render_404 if Rails.env.production? || !current_user.admin?
+
+    form_names = Foorm::Form.pluck(:name, :version)
+    formatted_names_and_versions = []
+    form_names.each do |form_data|
+      formatted_names_and_versions << {name: form_data[0], version: form_data[1]}
+    end
```

Comment:
nit: could be shorter
```suggestion
formatted_names_and_versions = Foorm::Form.all.map {|form| {name: form.name, version: form.version}}
```